src/platform/datapath_raw_xdp_win.c

```diff
@@ -942,6 +942,40 @@ CxPlatDpRawInitialize(
         return QUIC_STATUS_INVALID_PARAMETER;
     }
 
+    {
+        QuicTraceLogVerbose(
+            XdpInitialize,
+            "[ xdp][%p] XDP loading library",
+            Xdp);
+
+        HMODULE Lib = LoadLibraryA("xdpapi.dll");
```

Comment:
I thought we already had dynamic loading for xdp...?

Reply:
[This whole problem is moot with even newer builds.]